common/app/common/pages/PageHeader.java

```diff
@@ -0,0 +1,31 @@
+package common.pages;
+
+import common.cms.CmsPage;
+
+public class PageHeader {
```

Comment:
Then its more like a helper for producing well structured JSON for handlebars templates?